docs/src/__configuration__/markdownMapping.tsx

```diff
@@ -0,0 +1,269 @@
+    ),
+    blockquote: (props: TypographyProps<'blockquote'>): JSX.Element => {
+        return (
+            <Typography
+                component={'blockquote'}
+                sx={{
+                    pr: 2,
+                    mb: 1,
+                    borderLeft: ' 4px solid #DDDDDD',
+                    p: '0 15px',
+                    m: '15px 0',
+                    color: '#666666',
+                }}
+                {...props}
+            />
+        );
+    },
+    pre: (props: TypographyProps<'pre'>): JSX.Element => {
+        return (
+            <Typography
+                component={'pre'}
+                color={'textPrimary'}
+                sx={{
+                    pr: 2,
+                    mb: 1,
+                    display: 'flex',
+                    backgroundColor: (theme: Theme) =>
+                        theme.palette.mode === 'light' ? theme.palette.background.default : Colors.darkBlack[300],
+                }}
+                {...props}
+            />
+        );
+    },
+    code: (props: TypographyProps<'code'>): JSX.Element => {
+        return (
+            <Typography
+                component={'code'}
+                color={'textPrimary'}
+                sx={{
+                    fontSize: 14,
+                    m: '0px 2px',
+                    p: '0px 5px',
+                    whiteSpace: 'nowrap',
+                    border: '1px solid #EEEEEE',
+                    backgroundColor: '#F8F8F8',
+                    borderRadius: '3px',
+                    fontFamily: 'monospace',
+                    lineHeight: '1.6',
+                }}
+                {...props}
+            />
+        );
+    },
+    inlineCode: (props: TypographyProps<'code'>): JSX.Element => {
+        return (
+            <Typography
+                component={'code'}
+                color={'textPrimary'}
+                sx={{
+                    backgroundColor: (theme: Theme) =>
+                        theme.palette.mode === 'light' ? theme.palette.background.default : Colors.darkBlack[300],
+                    fontFamily: 'Roboto Mono, Monospaced',
+                    border: (theme: Theme) =>
+                        theme.palette.mode === 'light' ? undefined : `${theme.palette.divider} 1px solid`,
+                }}
+                {...props}
+            />
+        );
+    },
+    table: (props: TableProps): JSX.Element => {
+        return (
+            <Box
+                className="tableContainer"
+                sx={{
+                    overflow: 'auto',
+                    boxSizing: 'border-box',
+                    mb: 2,
+
+                    table: {
+                        p: '6px 13px',
+                        textAlign: 'left',
+                        borderCollapse: 'collapse',
+                        minWidth: 750,
+                        border: `1px solid ${Colors.black[100]}`,
```

Comment:
Sorry for not noticing this earlier. The table border color should be the same as `theme.palette.divider` — `rgba(66, 78, 84, 12%)`. Everything else looks good to me visually.